src/webServiceClient.spec.ts

```diff
@@ -434,17 +458,13 @@ describe('WebServiceClient', () => {
       });
     });
 
-    it('handles general http.request errors', () => {
+    it('handles general network errors', () => {
       const ip = '8.8.8.8';
-
-      const error = {
-        code: 'FOO_ERR',
-        message: 'some foo error',
-      };
```

Comment:
It wasn't clear to me where the value was in testing unexpected JSON via `replyWithError()` which seems to be well suited to mocking failed network requests -- a situation where no JSON at all may be returned. We are already testing that in the `handles errors with unknown payload` test.

Reply:
IIRC, it was to verify that the client could handle new maxmind error codes